java/src/main/java/com/adhoc/flight/utils/PrintTinyIntVectorProcessor.java

```diff
@@ -0,0 +1,65 @@
+package com.adhoc.flight.utils;
+
+import org.apache.arrow.vector.TinyIntVector;
+
+import com.adhoc.flight.client.BaseTinyIntVectorProcessor;
+
+public final class PrintTinyIntVectorProcessor extends BaseTinyIntVectorProcessor {
+
+	private long count;
+	
+	@Override
+	public long getCount() {
+		return count;
+	}
+
+	@Override
+	public void incrementCount() {
+		count++;
+	}
+
+	@Override
+	public String process(TinyIntVector tinyIntVector) {
+		byte[] bytes = new byte[tinyIntVector.getValueCount()];
```

Comment:
We should not be getting objects one-by-one like this. The idea was to use ArrowStreamWriter to dump the data vector-wise, perhaps into a ByteArrayInputStream.

Reply:
Solved in the new version.